src/DssSpell.sol

```diff
@@ -106,223 +57,129 @@ contract DssSpellAction is DssAction {
-            MedianAbstract(GNO_MEDIANIZER).kiss(SPARK_GNO_ORACLE);
-
-            // Set DAI as a borrowable asset in isolation mode
-            PoolConfiguratorLike(SPARK_POOL_CONFIGURATOR).setBorrowableInIsolation(MCD_DAI, true);
-
-            // Add GNO
-            address token = DssExecLib.getChangelogAddress("GNO");
-            PoolConfiguratorLike.InitReserveInput[] memory input = new PoolConfiguratorLike.InitReserveInput[](1);
-            input[0] = PoolConfiguratorLike.InitReserveInput({
-                aTokenImpl: SPARK_ATOKEN_IMPL,
-                stableDebtTokenImpl: SPARK_STABLE_DEBT_TOKEN_IMPL,
-                variableDebtTokenImpl: SPARK_VARIABLE_DEBT_TOKEN_IMPL,
-                underlyingAssetDecimals: GemAbstract(token).decimals(),
-                interestRateStrategyAddress: SPARK_INTEREST_RATE_STRATEGY,      // Dummy strategy - compare to other borrow-disabled asset like sDAI
-                underlyingAsset: token,
-                treasury: SPARK_TREASURY,
-                incentivesController: address(0),
-                aTokenName: "Spark GNO",
-                aTokenSymbol: "spGNO",
-                variableDebtTokenName: "Spark Variable Debt GNO",
-                variableDebtTokenSymbol: "variableDebtGNO",
-                stableDebtTokenName: "Spark Stable Debt GNO",
-                stableDebtTokenSymbol: "stableDebtGNO",
-                params: ""
-            });
-            PoolConfiguratorLike(SPARK_POOL_CONFIGURATOR).initReserves(input);
-            PoolConfiguratorLike(SPARK_POOL_CONFIGURATOR).configureReserveAsCollateral({
-                asset: token, 
-                ltv: 20_00,
-                liquidationThreshold: 25_00,
-                liquidationBonus: 110_00
-            });
-            PoolConfiguratorLike(SPARK_POOL_CONFIGURATOR).setDebtCeiling(token, 5 * MILLION * HUNDRED);
-
-            address[] memory tokens = new address[](1);
-            tokens[0] = token;
-            address[] memory oracles = new address[](1);
-            oracles[0] = SPARK_GNO_ORACLE;
-            AaveOracleLike(SPARK_AAVE_ORACLE).setAssetSources(
-                tokens,
-                oracles
-            );
-        }
-
-        // Reduce Maker Protocol GNO Debt Ceiling to Zero
-        (,,,uint256 line,) = VatLike(MCD_VAT).ilks("GNO-A");
-        DssExecLib.removeIlkFromAutoLine("GNO-A");
-        DssExecLib.decreaseIlkDebtCeiling("GNO-A", line / RAD, true);
-
-        // Bump the chainlog
-        DssExecLib.setChangelogVersion("1.14.12");
+        uint256 lineReduction;
+        uint256 line;
+
+        // --- BlockTower Vault Debt Ceiling Adjustments ---
+        // Poll: https://vote.makerdao.com/polling/QmPMrvfV#poll-detail
+        // Forum: https://forum.makerdao.com/t/blocktower-credit-rwa-vaults-parameters-shift/20707
+
+        // Decrease the Debt Ceiling (line) of BlockTower S1 (RWA010-A) from 20 million Dai to zero Dai.
+        DssExecLib.setIlkDebtCeiling("RWA010-A", 0);
```

Comment:
I think we also need to decrease globa DC for RWA10-A and RWA11-A

Reply:
https://docs.google.com/spreadsheets/d/1w_z5WpqxzwreCcaveB2Ye1PP5B8QAHDglzyxKHG3CHw/edit#gid=2128141805&range=F4

This claims that the block tower changes are net neutral. 

Also if you take a look at the current state (e.g. config): 
  - RWA010 got -20m 
  - RWA011 got -30m
  - RWA012 got +50m

Reply:
So it's rather the 3rd argument to `increseIlkDebtCeiling` for 12th which is currently `true` but should be false